server/src/repository/manage.rs

```diff
@@ -1,14 +1,18 @@
 use std::path::Path;
-use std::process::{Stdio};
-use anyhow::anyhow;
+use anyhow::{anyhow, Context};
 use tokio::process::Command;
+use crate::repository::crypto;
 
 fn db_file(name: &str) -> String {
     format!("{name}.db.tar.gz")
 }
 
 pub async fn add(name: &str, packages: &Vec<String>, dir: &Path) -> anyhow::Result<()> {
     let mut command = Command::new("repo-add");
+    
+    if crypto::should_sign_packages() {
+        command.args(["--verify", "--sign"]);
```

Comment:
We can't use these options, as we don't have added the secret key to the system gpg. In my case this triggers a prompt which then waits for a timeout.

We either don't sign the repos themselves, or sign them afterwards with our key manually.

Reply:
Now we sign the database afterwards manually (like we do sign packages)